BuildAndTest.proj

```diff
@@ -99,18 +99,29 @@
       <VsixExpInstallerArgs>/rootSuffix:RoslynDev /vsInstallDir:"$([System.IO.Path]::GetFullPath('$(MSBuildBinPath)\..\..\..'))"</VsixExpInstallerArgs>
     </PropertyGroup>
 
-    <Exec Condition="'$(TestVsi)' != 'true' AND '$(SkipCoreClr)' != 'true'" Command="&quot;$(CoreRunExe)&quot; $(CoreRunArgs)" />
+    <!-- Run CoreClr tests -->
+    <Exec Condition="'$(TestVsi)' != 'true' AND '$(SkipCoreClrTests)' != 'true'" 
```

Comment:
Both of these properties already existed. I'd be happy to unify, but I think that is out of the scope of this PR. Since we can't test netci in the PR I'm worried about even these changes failing. I'll file a bug to follow up on this.